ProcessExtensions/ProcessExtensions.cs

```diff
@@ -207,7 +254,12 @@ private static bool GetSessionUserToken(ref IntPtr phUserToken)
             return bResult;
         }
 
-        public static bool StartProcessAsCurrentUser(string appPath, string cmdLine = null, string workDir = null, bool visible = true)
+        public static bool StartProcessAsUser(string user, string appPath, string cmdLine = null, string workDir = null, bool visible = true)
+        {
+            return StartProcessAsCurrentUser(appPath, cmdLine, workDir, visible, user);
+        }
+
+        public static bool StartProcessAsCurrentUser(string appPath, string cmdLine = null, string workDir = null, bool visible = true, string user = null)
```

Comment:
It doesn't make sense for `StartProcessAsCurrentUser` to take a `user` parameter. I would reverse this relationship, so that `StartProcessAsCurrentUser` calls `StartProcessAsUser` (passing `null`).